visidata/features/errors_guide.py

```diff
@@ -0,0 +1,29 @@
+from visidata import GuideSheet, vd
+
+class ErrorsGuide(GuideSheet):
+    guide_text='''# A guide to errors in VisiData
+
+There are two severity levels VisiData will use when communicating something has gone wrong.
+If in performing an operation it reaches a [:166]warning[/], the operation will continue.
+With [:202]errors[/], operations will stop.
+
+[:202]Errors[/] and/or [:166]warnings[/] can manifest in different ways. How they appear determines how they are folllowed up on.
+
+1. [:202]a phrase all in lowercase[/] [:166]appears in the sidebar[/]
```

Comment:
Let's use semantic instead of numeric colors, and be more specific than "a phrase": `[:warning]an error message[/][:sidebar]appears in the sidebar[/]`

Reply:
Ooooo, so cool! I had no idea I could use these semantics. That's a lot better.